canonical/ght
# Added logout command

! Review this PR after [the login PR](https://github.com/canonical/ght/pull/84)
## Done
Logout command is added

## QA
- Checkout the feature branch
- Install dependencies with `yarn`
- Run `ts-node ./src/index.ts login`
- Verify ".canonical-greenhouse.json" is created
- Run `ts-node ./src/index.ts logout` 
- Verify ".canonical-greenhouse.json" is deleted
- Run `ts-node ./src/index.ts logout` again
- Verify no error is thrown

Fixes #80
